CMakeLists.txt

```diff
@@ -644,9 +643,10 @@ if(OPENSSL_FOUND)
 	target_sources(torrent-rasterbar PRIVATE src/pe_crypto)
```

Comment:
my understanding is that this line is a mistake. It's repeated on line 665, which is where I think it belongs. I think someone at some point mixed up using libcrypto and bittorrent protocol encryption

Reply:
this line has been removed in master recently too

Reply:
I must admit my problem was that I did not really look at the file and couldn't figure out what the file was for just from its name, but now it makes sense, **p**rotocol **e**ncryption, of course :). I looked at it now and yeah it does not depend on `libcrypto` (or `libgcrypt`) in any way as far as I can tell.

Fixed.